www/googlemaps-cdv-plugin.js

```diff
@@ -2632,10 +2632,13 @@ function getAllChildren(root) {
     var clickable;
     var style, displayCSS, opacityCSS, visibilityCSS, node, clickableSize;
 
-    var allClickableElements = Array.prototype.slice.call(root.querySelectorAll(':not([data-clickable="false"])'));
-    var clickableElements =  allClickableElements.filter(function(i) {return i != root;});
+    root = document.querySelector('.main-region.foreground');
```

Comment:
Oh, I didn't see that. In that case I guess I can just add something like a default selector to look for clickable elements in the `setDiv` method.